docs/modules/ROOT/pages/alias_catalogs.adoc

```diff
@@ -47,7 +47,7 @@ The full format is `<alias>@<user/org>(/repository)(/branch)(~path)` or `<alias>
 |Command | Description
 
 |`jbang hello@acme.corp`
-|`hello` alias found in `https://acme.corp/jbang-catalog.json` if available or the default branch searched on github, gitlab and bitbucket in that order.
```

Comment:
Why remove that info? Jbang does do that search.

Reply:
Really ? I though the '.' used indicated a catalog on a https site instead of of a Git backed service.

I should have missed something :-)

Reply:
I think you're actually right @grumpyf0x48 , that looks like a copy&paste error (or _I_ don't understand how it works either hehe)

Reply:
well, the code just have it all setup as a chain and can be seen here: https://github.com/jbangdev/jbang/blob/2dceffb51c799b6cba9bac8d5a449fd25b5e7810/src/main/java/dev/jbang/catalog/ImplicitCatalogRef.java#L72

basically we use https:// prefix OR a dot in host name to decide if https lookup should be done ....if it finds something all good; but if it fails it gets treated as a git repo lookup.